data/rashmip2408.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Rashmi Pawar",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Frontend developer **Beginner**. Like to read Books Yoga & Travel",
+  "avatar": "https://github.com/rashmip2408.png"
+  "links": [
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "linkedin.com/in/rashmi-pawar-b4888322b",
+      "icon": "FaLinkedIn"
```

Comment:
fix this LinkedIn icon, use `FaLinkedin`

Reply:
![image](https://user-images.githubusercontent.com/119780744/213932260-7d618422-87c8-4ff0-817f-5bb67857d06a.png)
commit directly to branch or create new branch again?

Reply:
same branch, yes this `patch-1`